Frontend/src/views/Sponsors.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div class="container-fluid">
+        <h3>Sponsoren des Jahres {{ new Date().getFullYear() }}</h3>
+    </div>
+    <div class="card" v-for="(sponsor, index) in sponsors" v-bind:key="sponsor.id">
+        <Sponsor
+            :sponsor="sponsor"
+            :linkMessage="linkMessages[Math.floor(Math.random() * linkMessages.length)]"
+        />
+    </div>
+</template>
+
+<script setup>
+import { ref } from "vue";
+import ApiService from "@/services/ApiService";
+import { shuffle } from "lodash"
+
+//Shuffle all entries but sort goldsponsors to the front
+const responseData = shuffle((await ApiService.getSponsors()).data).sort((a, b) => {
```

Comment:
hier würde ich erst den status prüfen. ansonsten crashed der bei nem error, weil in .data nix drin steht und der error wird geshuffelt:D